utils/ssl_renewel_util/certs_renew_cron.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Prompt the user for the domain name
+read -p "Enter the domain name: " mosip_domain
+
+expiry_date=$(sudo openssl x509 -enddate -noout -in /etc/letsencrypt/live/$mosip_domain/fullchain.pem)
```

Comment:
update openssl command

```
date --date="$(sudo openssl x509 -enddate -noout -in /etc/letsencrypt/live/dev.mosip.net/fullchain.pem | sed -e 's/notAfter=//g') -7 days" +'%d %m'
```

renew ssl certs before 7 day of expiry

Reply:
updated